test/CMakeLists.txt

```diff
@@ -120,8 +120,9 @@ endfunction()
 
 
 # Include generated files (moc ouput, build configuration)
-include_directories("${CMAKE_CURRENT_BINARY_DIR}/../src")
-include_directories("${CMAKE_CURRENT_BINARY_DIR}")
+include_directories("${PROJECT_BINARY_DIR}/src")
+include_directories("${PROJECT_BINARY_DIR}")
```

Comment:
I noticed ;-) Take a step back. We need to include:
- 123: files generated in build tree /src
- 124: files generated in build tree /test
- 125 (new): files in source tree /src

So when using PROJECT_XXX_DIR for /src in line 125 (src in source tree, "line inserted below"), I suggested to also use PROJECT_XXX_DIR for /src in line 123 (src in build tree).
For the current binary directory (line 124), there is no need to take the way over PROJECT_XXX_DIR.